group-coordinator/src/test/java/org/apache/kafka/coordinator/group/OffsetMetadataManagerTest.java

```diff
@@ -187,8 +190,6 @@ public CoordinatorResult<OffsetCommitResponseData, Record> commitOffset(
             short version,
             OffsetCommitRequestData request
         ) {
-            snapshotRegistry.getOrCreateSnapshot(lastCommittedOffset);
```

Comment:
What's the context behind removing this?

Reply:
The snapshot is already created in `replay` so we don't need it here.

Reply:
Ok -- so just a bug we didn't see before.